### PR TITLE
Fix: other_duties visibility

### LINKED ISSUE #3845: Prevent Admins from seeing other duties across orgs
### Impacted User Types
 - admins

### Environment
all

### Current Behavior
Admins see all "Other Duties" even from other casa orgs
image

### Expected Behavior
Admins see only other duties from their own org

### How to Replicate
ex: 
1. - Log in as a volunteer
2. - Create a new duty
3. - Log in as a volunteer from a different org
4. - Create a new duty
5. - Log in as any admin
6. - Go to the other duties page via the left nav bar
7. - See both duties and volunteers listed

### QA Login Details:
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- casa_admin2-1@example.com view the site as an admin from a different org

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by tonyaraujop**
### What github issue is this PR for, if any?
Resolves #3845

### What changed, and why?

Change query for volunteers to get only duties associated with admin.

### How will this affect user permissions?
- Volunteer permissions: N/A
- Supervisor permissions: N/A
- Admin permissions: Prevent Admins from seeing other duties across orgs

### How is this tested? (please write tests!) 💖💪

Tests were written with tww different contexts. One when admin is on the same org as volunteer and the other is when admin is on a different org than the volunteer.

### Screenshots please :)
When admin is on the same org as volunteer
image

When admin is on a different org than the volunteer
image

### CODE REVIEW

**Comment by compwron** [comment]
Lint is failing- other than that, looks good!

File: app/controllers/other_duties_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def edit
 
   def index
     @volunteer_duties = if current_user.casa_admin?
-      generate_other_duty_list(Volunteer.all)
+      generate_other_duty_list(Volunteer.where(casa_org_id: current_user.casa_org_id))
```
Comment (compwron) [inline app/controllers/other_duties_controller.rb:24]: the preferred way to do this is to use a `policy_scope` from https://github.com/varvet/pundit

Comment (tonyaraujop) [reply app/controllers/other_duties_controller.rb:24]: Sure, I will look into this.

**Comment by compwron** [comment]
Nice

**Pull request merged**